src/wallet/rpc/backup.cpp

```diff
@@ -401,6 +401,7 @@ RPCHelpMan importdescriptors()
             }
         }
         pwallet->ConnectScriptPubKeyManNotifiers();
+        pwallet->RefreshAllTXOs();
```

Comment:
In "wallet: Recalculate the wallet's txos after any imports" (850de8343e36a17a5c875b1be974153bf06efaca):
I’m surprised that the prior commit did not have failing tests if the addition of `RefreshAllTXOs()` is necessary here. Is there maybe test coverage missing here?

Reply:
The test coverage is added in the next commit "test: Test for balance update due to untracked output becoming spendable" which I have now moved to be earlier in the PR.